packages/keyring-snap-bridge/package.json

```diff
@@ -37,18 +37,24 @@
   "dependencies": {
     "@ethereumjs/tx": "^4.2.0",
     "@metamask/eth-sig-util": "^8.0.0",
+    "@metamask/keyring-api": "workspace:^",
+    "@metamask/keyring-internal-api": "workspace:^",
+    "@metamask/keyring-snap-internal-client": "workspace:^",
+    "@metamask/keyring-snap-sdk": "workspace:^",
```

Comment:
Do you know why the `sdk` is needed in the bridge? It feels odd.

Reply:
All good now, this was a mistake! Thanks :)